GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -347,6 +348,8 @@ private void Update()
 
             var commandStartTime = Environment.TickCount;
             _workTreeWatcher.EnableRaisingEvents = true;
+            _gitDirWatcher.EnableRaisingEvents = Directory.Exists(_gitDirWatcher.Path) &&
+                    !_gitDirWatcher.Path.StartsWith(_workTreeWatcher.Path);
```

Comment:
Nitpick: This code is an identical copy.

Reply:
A separate method will be (even if that hides the handling)